guides/source/active_storage_overview.md

```diff
@@ -464,6 +464,13 @@ HTTP expiration of 5 min.
 url_for(user.avatar)
 ```
 
+WARNING: The links generated by ActiveStorage are hard to guess, but publicly
+accessible by default. Anyone that knows the blob URL will be able to download it,
+even if a `before_action` in your `ApplicationController` would otherwise
+require a login. If your files require a higher level of protection consider
+implementing your own authenticated
+[redirection controller](https://github.com/rails/rails/blob/master/activestorage/app/controllers/active_storage/blobs/redirect_controller.rb).
```

Comment:
I am not sure, whether links to the source code are common at all. However, it would also seem weird to me to ask users to override a class without linking to its source code.

Reply:
There are other places, even within this document, that we link to source code for similar reasons so I think this is fine.


There are two places we should link to:

```suggestion
[`ActiveStorage::Blobs::RedirectController`](https://github.com/rails/rails/blob/master/activestorage/app/controllers/active_storage/blobs/redirect_controller.rb) and [`ActiveStorage::Representations::RedirectController`](https://github.com/rails/rails/blob/master/activestorage/app/controllers/active_storage/representations/redirect_controller.rb).
```